src/util/common.js

```diff
@@ -693,14 +695,14 @@ var util = {
       if (((p0.y > pt.y) !== (p1.y > pt.y)) && (pt.x < (p1.x - p0.x) * (pt.y - p0.y) / (p1.y - p0.y) + p0.x)) {
         inside = !inside;
       }
-      distSq = util.distance2dToLineSquared(pt, p0, p1);
+      distSq = onlySign ? 1 : util.distance2dToLineSquared(pt, p0, p1);
```

Comment:
so if onlySign is true, the function will return distSq == 1? I did not look at the entire block, just the diff.

Reply:
distSq will be 1 (because we don't care about the actual value of the distance), but the returned result will be -1 or +1 depending on the value of the `inside` flag.

Reply:
thanks makes sense.